conf/modules.config

```diff
@@ -241,12 +241,12 @@ process {
     }
 
     withName: PORECHOP_PORECHOP {
-        ext.prefix = { "${meta.id}_${meta.run_accession}" }
+        ext.prefix = { "${meta.id}_${meta.run_accession}_porechop" }
         publishDir = [
             [
                 path: { "${params.outdir}/porechop" },
                 mode: params.publish_dir_mode,
-                pattern: '*_porechopped.fastq.gz',
+                pattern: '*.fastq.gz',
```

Comment:
Is there any reason to remove the suffiex, when it's actually already defined with `ext.prefix`?